src/physics/grid/ValueGridStore.cc

```diff
@@ -0,0 +1,91 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file ValueGridStore.cc
+//---------------------------------------------------------------------------//
+#include "ValueGridStore.hh"
+
+#include "base/SpanRemapper.hh"
+#include "base/VectorUtils.hh"
+#include "comm/Device.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct with storage space requirements.
+ *
+ * This allocates device data and resizes it to empty (so that the vector can
+ * keep track of its actual size).
```

Comment:
Seems to just reserve host data?

Reply:
Quite so. The comment was from an earlier revision, before I separated on-host assembly from the device construction.